src/info/manifest.c

```diff
@@ -1649,11 +1656,11 @@ manifestLoadCallback(void *callbackData, const String *section, const String *ke
     else if (strEq(section, MANIFEST_SECTION_BACKUP_DB_STR))
     {
         if (strEq(key, MANIFEST_KEY_DB_ID_STR))
-            manifest->data.pgId = jsonToUInt(value);
+            manifest->data.pgId = varUIntForce(value);
         else if (strEq(key, MANIFEST_KEY_DB_SYSTEM_ID_STR))
-            manifest->data.pgSystemId = jsonToUInt64(value);
+            manifest->data.pgSystemId = varUInt64(value);
```

Comment:
Why does `manifest->data.pgId = varUIntForce(value);` (among others) require Force but `manifest->data.pgSystemId = varUInt64(value);` does not require `varUInt64Force(value); `?

Reply:
Since numbers are not typed in JSON we always convert them to uint64. So if you want uint you need to force it.